lib/shopify_api/session.rb

```diff
@@ -73,9 +73,10 @@ def initialize(url, token = nil, extra = {})
       self.extra = extra
     end
 
-    def create_permission_url(scope, redirect_uri = nil)
-      params = {:client_id => api_key, :scope => scope.join(',')}
+    def create_permission_url(scope, redirect_uri = nil, state: nil)
+      params = {client_id: api_key, scope: scope.join(',')}
```

Comment:
Seems like we're only using ruby 2+ now so we can switch off the old style hashes right?